modules/virt-installing-virtctl-client-web.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// install/virt-installing-virtctl.adoc
```

Comment:
virt/install/virt-installing-virtctl.adoc

Reply:
Nice one - I see this in the module I copied from so I will also fix it in the other install modules as well.